docs/en/02_Developer_Guides/10_Email/index.md

```diff
@@ -14,15 +14,43 @@ covers how to create an `Email` instance, customise it with a HTML template, the
 Silverstripe CMS provides an API over the top of the [SwiftMailer](http://swiftmailer.org/) PHP library which comes with an
 extensive list of "transports" for sending mail via different services. 
 
-Out of the box, Silverstripe CMS will use the built-in PHP `mail()` command via the `Swift_MailTransport` class. If you'd
-like to use a more robust transport to send mail you can swap out the transport used by the `Mailer` via config:
+For legacy reasons, Silverstripe CMS 4 defaults to using the built-in PHP `mail()` command via a deprecated class `Swift_MailTransport` class. However, using this layer is less secure and is strongly discouraged.
```

Comment:
doubled-up word 'class' around the `Swift_MailTransport`

Reply:
Updated